packages/expo-router/src/layouts/withLayoutContext.tsx

```diff
@@ -75,6 +75,32 @@ export function useFilterScreenChildren(
 /**
  * Returns a navigator that automatically injects matched routes and renders nothing when there are no children.
  * Return type with `children` prop optional.
+ * 
+ * Enables use of other built-in React Navigation navigators and other navigators built with the React Navigation custom navigator API.
+ *
+ *  * @example
+ * ```_layout.tsx
```

Comment:
````suggestion
 * ```tsx app/\_layout.tsx
````

The language syntax is missing. Also, since it's referring to the file path, we can define a directory where this file is stored (for the example).